.goreleaser.yaml

```
@@ -0,0 +1,367 @@
+archives:
+  - id: archive-unix
+    format: binary
+    builds:
+      - binary-linux
+      - binary-darwin
+    # NB: specifically crafted to ensure compatibility with release artifacts < v3.8.0.
+    name_template: '{{ .ProjectName }}-v{{ .Version }}.{{ .Os }}.{{ .Arch }}'
+
+  - id: archive-windows
+    format: binary
+    builds:
+      - binary-windows
+    # NB: specifically crafted to ensure compatibility with release artifacts < v3.8.0.
+    name_template: '{{ .ProjectName }}-v{{ .Version }}'
+
+  - id: archive-darwin-universal
+    format: binary
+    builds:
+      - binary-darwin-universal
+    # NB: specifically crafted to ensure compatibility with release artifacts < v3.8.0.
+    # We can't bundle this with the other unix archive, because .Arch becomes "all".
+    # Before v3.8.0, this used to be _just_ the AMD64 binary.
+    name_template: '{{ .ProjectName }}-v{{ .Version }}.darwin'
+
+# xref: https://goreleaser.com/customization/checksum/
+checksum:
+  name_template: "{{ .ProjectName }}-v{{ .Version }}.checksums.txt"
+  algorithm: sha256
+  ids:
+    - archive-unix
+    - archive-windows
+    - archive-darwin-universal
+
+# xref: https://goreleaser.com/customization/sbom/
+sboms:
+  - id: binary-sbom
+    artifacts: binary
+    documents:
+      - "{{ .ArtifactName }}.spdx.sbom.json"
+
+# xref: https://goreleaser.com/customization/sign/
+signs:
+  - cmd: cosign
+    artifacts: checksum
+    signature: '{{ trimsuffix .Env.artifact ".txt" }}.sig'
+    certificate: '{{ trimsuffix .Env.artifact ".txt" }}.pem'
+    args:
+      - "sign-blob"
+      - "--output-signature"
+      - "${signature}"
+      - "--output-certificate"
+      - "${certificate}"
+      - "${artifact}"
+    output: true
+
+# xref: https://goreleaser.com/customization/docker/
+dockers:
+  - image_templates:
+      - 'ghcr.io/{{ .Env.GITHUB_REPOSITORY }}:v{{ .Version }}-amd64'
+      - 'quay.io/{{ .Env.GITHUB_REPOSITORY }}:v{{ .Version }}-amd64'
+    use: buildx
+    goos: linux
+    goarch: amd64
+    ids:
+      - binary-linux
+    dockerfile: .release/Dockerfile
+    build_flag_templates:
+      - "--pull"
+      - "--platform=linux/amd64"
+      - "--label=org.opencontainers.image.created={{ .Date }}"
+      - "--label=org.opencontainers.image.name={{ .ProjectName }}"
+      - "--label=org.opencontainers.image.revision={{ .FullCommit }}"
+      - "--label=org.opencontainers.image.version={{ .Version }}"
+      - "--label=org.opencontainers.image.source={{ .GitURL }}"
+
+  - image_templates:
+      - 'ghcr.io/{{ .Env.GITHUB_REPOSITORY }}:v{{ .Version }}-arm64'
+      - 'quay.io/{{ .Env.GITHUB_REPOSITORY }}:v{{ .Version }}-arm64'
+    use: buildx
+    goos: linux
+    goarch: arm64
+    ids:
+      - binary-linux
+    dockerfile: .release/Dockerfile
+    build_flag_templates:
+      - "--pull"
+      - "--platform=linux/arm64"
+      - "--label=org.opencontainers.image.created={{ .Date }}"
+      - "--label=org.opencontainers.image.name={{ .ProjectName }}"
```

I _think_ this should be 
```suggestion
      - "--label=org.opencontainers.image.title={{ .ProjectName }}"
```

if were following the  [OCI pre-defined keys](https://github.com/opencontainers/image-spec/blob/main/annotations.md)

---

You are absolutely right, great catch :bow: